SwiftLint/project-swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+###
+# .project-frameworks.yml
```

Comment:
What is this file for?
```suggestion
# .project-swiftlint.yml
```
?

Reply:
just the name of the file

Reply:
But what is the file for?
Is it copied to each the project?
What is supposed to be put in there?

Reply:
It's the template for the project specific configuration of swiftlint.
It is copied to the main repo if the file does not exist yet.
As explained in the description of this PR:

> Project Config Template
> When a project does not have a .project-swiftlint.yml file the script will now copy the (new) template to the 'parent' repo.

Reply:
Ah. Okay. Now I read the PR description.